src/commands/Google/weather.ts

```diff
@@ -20,7 +20,7 @@ const COLORS = {
 	windy: '#33B679'
 };
 
-const celsiusToFahrenheit = (celsius: number) => (celsius * 5) / 9 + 32;
+const celsiusToFahrenheit = (celsius: number) => celsius * (9 / 5) + 32;
```

Comment:
```suggestion
const celsiusToFahrenheit = (celsius: number) => (celsius * 9 / 5) + 32;
```

![image](https://user-images.githubusercontent.com/4019718/94652072-c333db80-02f9-11eb-904d-5f6b1f711128.png)


Same what I showed Stitch at the time